solutions/instances/version.tf

```diff
@@ -10,5 +10,9 @@ terraform {
       source  = "logdna/logdna"
       version = "1.16.0"
     }
+    time = {
+      source  = "hashicorp/time"
+      version = ">= 0.9.1, < 1.0.0"
```

Comment:
DAs need to lock into specific versions. Please lock into latest version of this provider